src/System.IO.Pipes.AccessControl/tests/AnonymousPipeTests/AnonymousPipeServerStreamAclTests.cs

```diff
@@ -0,0 +1,138 @@
+﻿using System.Collections.Generic;
+using System.Linq;
+using System.Security.AccessControl;
+using System.Security.Principal;
+using Xunit;
+
+namespace System.IO.Pipes.Tests
+{
+    public class AnonymousPipeServerStreamAclTests : PipeServerStreamAclTestBase
+    {
+        [Fact]
+        public void Create_NullSecurity()
+        {
+            CreateAndVerifyAnonymousPipe(PipeDirection.In, HandleInheritability.None, 0, null).Dispose();
+        }
+
+        [Theory]
+        [InlineData(PipeDirection.InOut)]
+        [InlineData((PipeDirection)(int.MinValue))]
+        [InlineData((PipeDirection)0)]
+        [InlineData((PipeDirection)4)]
+        [InlineData((PipeDirection)(int.MaxValue))]
+        public void Create_InvalidPipeDirection(PipeDirection direction)
+        {
+            if (direction == PipeDirection.InOut)
+            {
+                Assert.Throws<NotSupportedException>(() =>
+                {
+                    CreateAndVerifyAnonymousPipe(direction, HandleInheritability.None, 0, GetBasicPipeSecurity()).Dispose();
+                });
+            }
+            else
+            {
+                Assert.Throws<ArgumentOutOfRangeException>(() =>
+                {
+                    CreateAndVerifyAnonymousPipe(direction, HandleInheritability.None, 0, GetBasicPipeSecurity()).Dispose();
+                });
+            }
```

Comment:
I think it would be better to include the exception type in the InlineData, e.g.
```C#
[Theory]
[InlineData(PipeDirection.InOut, typeof(NotSupportedException))]
[InlineData((PipeDirection)int.MinValue, typeof(ArgumentOutOfRangeException))]
...
public void Create_InvalidPipeDirection(PipeDirection direction, Type expectedException)
{
    Assert.Throws(expectedException, () => CreateAndVerifyAnonymousPipe(direction, HandleInheritability.None, 0, GetBasicPipeSecurity()).Dispose());
}
```